app/assets/javascripts/components/hero-pool-form.jsx

```diff
@@ -37,31 +37,33 @@ class HeroPoolForm extends React.Component {
 
   loadHeroPool() {
     const api = new OverwatchTeamCompsApi()
-
     api.getHeroPool().
       then(heroPool => this.onHeroPoolLoaded(heroPool)).
       catch(err => HeroPoolForm.onHeroPoolFetchError(err))
   }
 
   render() {
-    const { heroes } = this.state
+    const { heroes, ranks } = this.state
     if (heroes.length < 1) {
```

Comment:
Not related to these changes, but I was curious if there was a convenient `empty` function defined on the `Array` prototype in ECMA 2016 JS. I wasn't able to find anything in the ECMA 2016 specific `Array` functions, but saw this:

```javascript
if (!!heroes.length) {
  ...
}
```

Because if `length` returns `0`, which is considered `false` as a Boolean conditional value, then this evaluates to `true` if `length > 0`. I think the existing conditional here is just fine and not suggesting a change, but thought this strategy was interesting to share.